packages/eas-json/src/utils.ts

```diff
@@ -73,4 +93,36 @@ export class EasJsonUtils {
     const easJson = await accessor.readAsync();
     return resolveSubmitProfile({ easJson, platform, profileName });
   }
+
+  public static getCacheSettings<T extends Platform>(
```

Comment:
just moving this function here does not solve the issue I described in my other comment, want getBuildProfileAsync already returns the converted build profile. 

Also you should not need anything from eas-cli package here.

Reply:
As mentioned above, I recommend doing that using only Joi schema without adding any procedural code.